app/views/accounts/_connected_app.html.erb

```diff
@@ -20,10 +20,12 @@
     <strong>
       <%= identity.email_address&.email || t('account.connected_apps.email_not_selected') %>
     </strong>
```

Comment:
We also don't want to show the "selected" email if the connected application didn't request email.

From the ticket:

>"No email selected" / selected email [...] are not shown

Reply:
Making sure I understand the changes to be made--so it should look like this:

<img width="585" alt="image" src="https://github.com/user-attachments/assets/b79b49c1-e673-47fc-8bde-082121208a03" />

Reply:
I think we'd want to fix up the stray "with:" there. I think this changed in the pull request which originally added the feature, so we might be able to get some strings from the previous implementation to use.

It might also be worth double-checking with the UX folks, but other than that, I think it's what I'd expect.

Reply:
That makes sense.